src/main/resources/swagger/api-docs.yaml

```diff
@@ -1162,6 +1161,24 @@ paths:
             - https://www.googleapis.com/auth/devstorage.full_control
             - https://www.googleapis.com/auth/cloud-billing
 
+  /api/profile/preferences:
```

Comment:
Not your fault, but this file is over 4k lines. That is unacceptable. It needs to be divided up and, ideally, each section moved as close as possible to the code that implements the documented functionality.

https://broadinstitute.atlassian.net/browse/GAWB-1911